seatunnel-connectors/seatunnel-connector-spark-clickhouse/src/main/scala/org/apache/seatunnel/spark/sink/Clickhouse.scala

```diff
@@ -16,37 +16,56 @@
  */
 package org.apache.seatunnel.spark.sink
 
-import java.math.BigDecimal
+import net.jpountz.xxhash.{XXHash64, XXHashFactory}
```

Comment:
@BenJFan Could you please talk about the purpose of using `XXHash64` ?

Reply:
When sharding key isn't a number, we need use hash to get which shard we should write.
``` scala
val offset = (hashInstance.hash(ByteBuffer.wrap(row.getString(fieldIndex).getBytes), 0) & Long.MaxValue %
                this.shardWeightCount).toInt
```

Reply:
@BenJFan I don't know much about `XXHash64` , is there any performance improvement for `XXHash64` using this? Why not use `Hash` in `jdk`

Reply:
This is benchmarks: https://cyan4973.github.io/xxHash/. Faster than MD5 etc.